include/Zydis/Formatter.h

```diff
@@ -1046,15 +1056,16 @@ ZYDIS_EXPORT ZyanStatus ZydisFormatterFormatInstruction(const ZydisFormatter* fo
  * @return  A zyan status code.
  */
 ZYDIS_EXPORT ZyanStatus ZydisFormatterFormatInstructionEx(const ZydisFormatter* formatter,
-    const ZydisDecodedInstruction* instruction, char* buffer, ZyanUSize length,
-    ZyanU64 runtime_address, void* user_data);
+    const ZydisDecodedInstruction* instruction, const ZydisDecodedOperand* operands,
+    ZyanU8 operand_count, char* buffer, ZyanUSize length, ZyanU64 runtime_address,
+    void* user_data);
 
 /**
  * Formats the given operand and writes it into the output buffer.
  *
  * @param   formatter       A pointer to the `ZydisFormatter` instance.
  * @param   instruction     A pointer to the `ZydisDecodedInstruction` struct.
- * @param   index           The index of the operand to format.
+ * @param   operand         A pointer to the `ZydisDecodedOperand` struct of the operand to format.
```

Comment:
Might be easier to understand like this:

```suggestion
 * @param   operand         A pointer to the array of operands.
```

Reply:
Ah, sorry -- meant to comment that on line 1027 / 1046, but selected the wrong one when switching between different diffing views. This change wouldn't make sense for this one.